src/components/Button/Button.js

```diff
@@ -33,17 +31,50 @@ const Button = ({ className, largeText, size, ...other }) => {
   );
 };
 
+/* eslint-disable react/require-default-props */
 Button.defaultProps = {
```

Comment:
- [ ] As we build on top of Carbon's `Button`, these `defaultProps` should reflect Carbon's, with the exception of `largeText`, which should remain as-is - https://github.com/carbon-design-system/carbon/blob/master/packages/react/src/components/Button/Button.js#L204